.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
Please resolve merge conflicts in 
.gitignore
src/crawler/config.yaml

Reply:
Conflicts merged